boards/arm64/fvp_baser_aemv8r/fvp_baser_aemv8r_smp.yaml

```diff
@@ -5,8 +5,11 @@ identifier: fvp_baser_aemv8r_smp
 name: FVP Emulation FVP_BaseR_AEMv8R
 arch: arm64
 type: sim
+simulation: armfvp
```

Comment:
@stephanosio  Can we remove this one, and skip `fvp_baser_aemv8r_smp` at this time. I may need more time to fix the SMP board. I can add it back when I fix all the test cases for `fvp_baser_aemv8r_smp`.

Reply:
Since this is not urgent, it can wait until `fvp_baser_aemv8r_smp` is fixed.